robottelo/ui/locators/model.py

```diff
@@ -253,11 +253,7 @@ def __repr__(self):
         if keys:
             msg.append("contains=%s" % str(keys))
         msg.append(">")
-        repr_text = u"|".join(msg)
-        if PY3:
-            return repr_text
-        else:
-            return repr_text.encode('utf-8')
+        return u"|".join(msg)
```

Comment:
https://docs.python.org/2/reference/datamodel.html#object.__repr__  , if you have not defined \__str\__ \__repr\__ will be used as str that is not unicode in 2.7